db/migrate/20190430212446_create_books.rb

```diff
@@ -0,0 +1,14 @@
+class CreateBooks < ActiveRecord::Migration[5.1]
+  def change
+    create_table :books do |t|
+      t.string :genre, :null => false, :default => Faker::Book.genre
```

Comment:
Why use a faker default? 🤔

Reply:
just to do not left this empy

Reply:
Do you think that this has sense? If I save a book without an author, you add a fake author? 🤔

Reply:
you're right sounds bad idea